sympy/functions/special/tests/test_hyper.py

```diff
@@ -394,3 +395,7 @@ def test_eval_nseries():
     assert hyper((1,2), (1,2,3), x**2)._eval_nseries(x, 7, None) == 1 + x**2/3 + x**4/24 + x**6/360 + O(x**7)
     assert exp(x)._eval_nseries(x,7,None) == hyper((a1, b1), (a1, b1), x)._eval_nseries(x, 7, None)
     assert hyper((a1, a2), (b1, b2), x)._eval_nseries(z, 7, None) == hyper((a1, a2), (b1, b2), x) + O(z**7)
+
+def test_aseries():
+    ex=integrate(1/sqrt(1+x**3),x)
+    assert ex.series(x,oo,2).evalf() == gamma(1/6)*gamma(1/3)/(3*sqrt(pi)) + sqrt(1/x)*gamma(-1/6)/(3*gamma(5/6)) + O(x**(-2), (x, oo))
```

Comment:
You should use e.g. `S(1)/6` rather than `1/6` to get an exact rational rather than a float.